src/httpserver/core.clj

```diff
@@ -1,6 +1,18 @@
-(ns httpserver.core)
+(ns httpserver.core
+  (:require [httpserver.handlers.teapot :as myteapot]
+            [httpserver.utilities.arguments :as argumenetParser])
+  (:gen-class))
+
+(import '(com.td.HttpServer HttpServer))
+(import '(com.td.HttpServer HttpResponse))
+(import '(com.td.HttpServer IHandler))
 
 (def display_welcome "Http Server!")
 
-(defn -main []
-  (println display_welcome))
```

Comment:
Looks like you can remove the `(def display_welcome "")` line above this as well.